doc/rst/language/spec/records.rst

```diff
@@ -519,8 +519,13 @@ following signatures for a record ``R``:
    proc ==(lhs:R, rhs:R) : bool where lhs.type == rhs.type;
    proc !=(lhs:R, rhs:R) : bool where lhs.type == rhs.type;
 
-Each of these compare the fields, one at a time, returning ``false`` if
-the property is not satisfied by the given pair of fields.
+Other comparison operator overloads, namely ``<``, ``<=``, ``>``, and ``>=`` have
+similar signatures but their where clauses also check whether the relevant
+operator is supported by each field.
+
+All except ``!=`` compare the fields, one at a time, returning ``false`` if
```

Comment:
I'd change "All except" to "All of these comparison operators except", since it is the start of a new paragraph.